ogr/services/pagure/pull_request.py

```diff
@@ -150,10 +150,11 @@ def create(
         }
 
         caller = project
-        if project.is_fork:
+        if project.is_fork and caller.parent:
```

Comment:
tbf, this could break tests too, because you need to query both `is_fork` and `parent`, and on the background it may be 2 separate API calls, even though one is enough :/

Reply:
Yes, this breaks something too IIRC.